### PR TITLE
Monkeys are now very rarely ghost roles

### PR DESCRIPTION
**Pull request by GoodWheatley**
## About the PR

This right now is effectively a free respawn for players, changing the probability so it only rarely happens retains the novelty of a sentient monkey without making it an easy way to self-antag.

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- remove: Regular monkeys are now very rarely available for ghosts to control.

### CODE REVIEW

**Comment by deltanedas** [comment]
not to mention on lrp the amount of shitters taking it as a free self antag role

**Comment by KaiShibaa** [comment]
History has repeated itself

**Comment by GoodWheatley** [comment]
PR updated

**Comment by metalgearsloth** [comment]
Shouldn't you just map fewer monkeys or nerf them? Probability ghost roles are cringe.

**Comment by Emisse** [comment]
i love monky

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 1]
```diff
@@ -814,7 +815,7 @@
       groups:
         Burn: 3
     clumsySound:
-      path: /Audio/Animals/monkey_scream.ogg
+      path: /Audio/Animals/monkey_scream.ogg
```
Comment (0x6273) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:818]: You messed up formatting. Revert this line the way it was.

**Review by 0x6273 — CHANGES_REQUESTED** [review]

**Comment by metalgearsloth** [comment]
fuck it

**Pull request merged**